src/libraries/System.IO.FileSystem/tests/Directory/Move.cs

```diff
@@ -239,6 +239,7 @@ public void ToNewDirectoryButWithDifferentCasing()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Casing differentiation bug fixed in .NET Core")]
```

Comment:
Consider including a link to the PR in the message.

Reply:
Sure. I'll look for the PRs.